contracts/src/interfaces/IUBIPool.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: GPL-3.0-only
+pragma solidity ^0.8.23;
+
+/// @title IUBIPool
+/// @notice Interface for the UBI Pool contract
+interface IUBIPool {
+    /// @notice Emitted when the UBI percentage is set
+    /// @param percentage The percentage of the UBI
+    event UBIPercentageSet(uint32 percentage);
+
+    /// @notice Emitted when the UBI distribution is set
+    /// @param month The month of the distribution (counter, starting from 1, not timestamp or month of year)
+    /// @param totalUBI The total amount of UBI
+    /// @param validatorUncmpPubKeys The validator uncompressed public keys
+    /// @param percentages The percentages of the UBI for each validator
+    event UBIDistributionSet(uint256 month, uint256 totalUBI, bytes[] validatorUncmpPubKeys, uint256[] percentages);
```

Comment:
it would be better to rename `percentages` to `amounts` to be consistent with implementation contract.

Reply:
Done, thanks for noticing!